terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,71 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                  = "filing-maintain" # this must match the stack name the service deploys into
+  name_prefix                 = "${local.stack_name}-${var.environment}"
+  global_prefix               = "global-${var.environment}"
+  service_name                = "limited-partnerships-api"
+  container_port              = "8080"
+  eric_port                   = "10000"
+  docker_repo                 = "limited-partnerships-api"
+  kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority   = 23
+  lb_listener_paths           = ["/limited-partnership*"]
```

Comment:
["/limited-partnership*"] this will also pick up the web requests which are /limited-partnerships/
suggest using ["/limited-partnership/*"]